sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -24,17 +30,21 @@
      * @param colno Column number.
      * @param absPath Absolute path.
      * @param platform Platform name.
+     * @param vars Local variables.
      */
+    // CHECKSTYLE.OFF: ParameterNumber
     public SentryStackTraceElement(String module, String function, String fileName, int lineno,
-                                   Integer colno, String absPath, String platform) {
+                                   Integer colno, String absPath, String platform, Map<String, Object> vars) {
```

Comment:
This is such an internal class (I suppose not enforced at the actual Java level) that it didn't/doesn't seem necessary to me.